src/main/kotlin/no/nav/syfo/aktivitetskrav/domain/Aktivitetskrav.kt

```diff
@@ -140,7 +141,7 @@ internal fun Aktivitetskrav.updateStoppunkt(oppfolgingstilfelle: Oppfolgingstilf
     )
 }
 
-internal fun Aktivitetskrav.vurder(
+fun Aktivitetskrav.vurder(
```

Comment:
Trenger denne være public?

Reply:
Tja.. Vi ville bruke samme logikk i repository-testen, så vi kaller på den der 🤔
Kan hende det er dårlig praksis å åpne opp bare for en test?